content/news/rsoc-arm64-0x01.md

```diff
@@ -0,0 +1,143 @@
++++
+title = "RSoC: Porting Redox to AArch64 - 0x01"
+author = "wizofe"
+date = "2018-05-30T18:33:21+00:00"
++++
+
+# Introductory session
+
+...and here we go! :floppy_disk: All excited. A first calendar entry to describe my attempt on ARM64 support in Redox OS. 
```

Comment:
Unfortunately, this (`:floppy_disk:`) doesn't seem to be handled by the markdown translator used on the website.